framework/src/parser/MooseSyntax.C

```diff
@@ -99,6 +99,7 @@ associateSyntax(Syntax & syntax, ActionFactory & action_factory)
 
   registerSyntax("AddOutputAction", "Outputs/*");
```

Comment:
You mean you want to do a `registerActionSyntax` followed by `registerSyntaxType` in a single macro?